app/celer-g4/CMakeLists.txt

```diff
@@ -130,6 +130,7 @@ endfunction()
 
 celer_app_test("gpu")
 celer_app_test("cpu")
+celer_app_test("cpu-nonfatal")
```

Comment:
Is the testing cpu specific or is the feature cpu specific?

Reply:
Neither :) The test adds 100% coverage to the new code.